packages/forklift-console-plugin/webpack.config.ts

```diff
@@ -97,6 +98,21 @@ const config: WebpackConfiguration & {
     devMiddleware: {
       writeToDisk: true,
     },
+    proxy: {
+      context: () => true,
+      target: 'http://localhost:30080/',
```

Comment:
Hardwired to a console deployed on a kind cluster?

Reply:
Yes. This matches https://github.com/kubev2v/forklift-console-plugin/blob/d0a32c380461dd3315070bb1fd6165e7c92786d6/ci/yaml/okd-console.yaml#L117